wnw.py

```diff
@@ -0,0 +1,171 @@
+"""Utilities for the word-nonword study"""
+
+import os.path as op
+
+
+WNW_DEFAULT_ROOT = "/data/NIMH_SFIM/handwerkerd/DenoisingPilot"
+WNW_SUBJECTS = [1, 3, 4]
+WNW_MODALITIES = ["func", "anat", "T1w"]
+WNW_RUNS = [1, 2]
+WNW_ACQUISITIONS = ["a", "b", "c"]
```

Comment:
Consider taking hardcoded paths and whatnot out to a environment setup script outside.  Because everyone's environments are usually different.

Reply:
Perhaps us os.environ with dotenv if you don't mind adding an external dependency.

Reply:
This is something I'm willing to consider, but I wanted to keep things specific to this project. It would be significantly better to generalize with something like that, though.

Reply:
Yes, I made that suggestion thinking of generalizing.  You're right that if that's not necessary, you're all set.

Reply:
![review](https://user-images.githubusercontent.com/26722533/144466792-f3e6c4ea-b623-496d-a366-134327ba3add.gif)